extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -128,7 +133,84 @@ public RepositoryAdapter build() {
       return new RepositoryAdapter(this);
     }
 
+    /**
+     * Builds the {@link RepositoryAdapter} that presents the provided repositories in order and
+     * observes the repositories as well as any additional observables while the provided
+     * {@link Activity} is active (between onResumed/onPaused).
+     */
+    @TargetApi(14)
+    @NonNull
+    public Adapter<ViewHolder> whileActive(@NonNull final Activity activity) {
```

Comment:
https://developer.android.com/reference/android/app/Activity.html says;
"If an activity is in the foreground of the screen (at the top of the stack), it is _active_ or _running_."
Doesn't _active_ sound better than _resumed_? Resumed doesn't really say.. anything...